source/includes/aws/_buckets.md

```diff
@@ -66,13 +124,25 @@ curl -X POST \
 }
 ```
 
-<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets/</code>
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/buckets&operation=create</code>
+
+Create a bucket in Amazon S3.
+
+| Required                   | &nbsp;                                        |
+|----------------------------|-----------------------------------------------|
+| `access` <br/>*string*     | The desired access level of the bucket. The list of supported Canned ACL is: private, public-read, public-read-write, authenticated-read, log-delivery-write, aws-exec-read.     |
+| `name` <br/>*string*       | The name of the bucket.                       |
+| `region` <br/>*string*     | The region to create the bucket in.           |
 
 | Attributes                 | &nbsp;                                        |
 |----------------------------|-----------------------------------------------|
-| `taskId` <br/>*string*     | The task id related to the bucket creation. |
+| `taskId` <br/>*string*     | The task id related to the bucket creation.   |
 | `taskStatus` <br/>*string* | The status of the operation.                  |
 
+| Mandatory Query Parameters | &nbsp;                                        |
+|----------------------------|-----------------------------------------------|
+| `operation` <br/>*string*  | The type of operation to execute. In this case, `create`. |
+
```

Comment:
```suggestion
```
we dont usually specify this - partially because it will always be create for this create operation - and its already specified in the post URL